src/threat_model/threat_model.md

```diff
@@ -324,18 +324,21 @@ cleared.
-| *10* | *A set of mutually trusted clients has restricted read-write access to the service IPC endpoint.*                                                                  |
+Most of these operational mitigations are implemented when following the [Recommendations on a
+Secure Parsec Deployment guide](secure_deployment.md).
+
+| ID   | Justification                                                                                                                                                      | Details                                                                                                                                                  |
+|------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0    | A trusted administrator should start the service in such a way that un-privileged processes cannot send signals to the service.                                    | In a Unix OS, using the separate `parsec` user to run the service prevents that.                                                                         |
+| 1    | A trusted administrator should check the validity of the providers and hardware modules written in the service's configuration before starting/reloading it.       | This is a manual step that has to be done as part of configuring the service.                                                                            |
+| 2    | Clients need to know from a trusted source that a trusted Parsec service is running on their machine so that they can trust the Listener endpoint.                 | The endpoint folder is owned by the `parsec` user, which must have been created by the trusted privileged administrator. This can be checked by clients. |
+| 3    | Mappings should be put in a location that only the Parsec service and the trusted administrator can access.                                                        | The mappings are inside the `parsec` user home folder.                                                                                                   |
+| 4    | The trusted administrator needs to be notified when any kind of resource is running out.                                                                           | Currently only the logs can be used as a way of knowing that the system is failing because a resource is running out.                                    |
+| 5    | Parsec logs coming from the service binary should be redirected to a file that is only writable by the service and readable by the trusted administrator.          | Using systemd, the logs can only be written by the service and read by the `parsec` user. Otherwise, they can be stored in the `parsec` home folder.     |
```

Comment:
Are you sure the part about "can only be read by the `parsec` user" is factual? On my machine, at least, everyone in the `adm` group can read them.

Reply:
I remember I tried it once and I couldn't read the service logs with `journalclt --user -u parsec` when logged as another user.
I guess there are privileged users or groups that can still read them but then this is implied by assumption 4:

> Users with privilege rights are trusted to exercise them in a non-malicious way.